utils/getUrl.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * Utility file to retrieve urls from the urls.json file.
+ */
+
+import urls from '../public/urls.json'
+
+// Type alias for the keys in urls.json
+export type UrlType = keyof typeof urls
+
+/**
+ * Get url from urls.json
+ */
+export default function getUrl(key: UrlType): string | undefined {
```

Comment:
I don't know how we want to handle this, and we could wait to refactor until it we have the actual different URLs, but I believe a bunch of EDD URLs are different for English vs Spanish (like not a query parameter, just a different URL for the same page)

Reply:
those URLs can live in the locale files!